code/lib/cli/src/automigrate/fixes/add-react.ts

```diff
@@ -49,11 +49,11 @@ export const addReact: Fix<AddReactOptions> = {
     return dedent`
       We've detected that you're using ${dependentsFormatted}.
       
-      Starting in Storybook 7 we now require the following peer dependencies:
+      Starting in Storybook 7, we now require these peer dependencies to render MDX docs:
```

Comment:
Does that mean that people who don't have mdx but used docs tab (autodocs now) don't need the react deps?